### PR TITLE
Wip/bhahn/BSPIMX8M-3179: cleanup mainline head for release

### PR DESCRIPTION
**Pull request by BHahn42618**
remove chapters about unsupported features:
- no SPI NOR Flash support
- no suspend to RAM support
- no support for EEPROM ID-Page
- no support for audio

### CODE REVIEW

File: source/bsp/imx8/imx8mp/mainline-head.rst [hunk 1]
```diff
@@ -816,8 +816,11 @@ EEPROM
 ------
 
 There are two different i2c EEPROM flashes populated on |som| SoM and on the
-|sbc|. Both can be used with the sysfs interface in Linux. The ID page of the
-I2C EEPROM populated on the SoM is also used for board detection.
+|sbc|. Both can be used with the sysfs interface in Linux.
+
+.. note::
+   There is no support for the ID page of the I2C EEPROM populated on the SoM yet,
+   but usually (in other releases) it will also be used for board detection.
```
Comment (tremmet) [inline source/bsp/imx8/imx8mp/mainline-head.rst:823]: The ID page **is** used for board detection. See: https://github.com/phytec/u-boot-phytec/blob/30ed1d3bc40f724af68e83a027e357af657bdd0d/board/phytec/phycore_imx8mp/spl.c#L29 The only thing is that you can not read write the data with a EEPROM driver in U-Boot and kernel as the device is not included in the device tree. So you should rephrase this a little bit.

**Review by tremmet — CHANGES_REQUESTED** [review]

File: source/bsp/imx8/imx8mp/mainline-head.rst (hunk 1 above)
Comment (BHahn42618) [reply source/bsp/imx8/imx8mp/mainline-head.rst]: Ok, done.

**Comment by ymoog** [comment]
@BHahn42618 What is the goal of this PR? Please explain that in the PR summary comment.

**Review by ymoog — DISMISSED** [review]

File: source/bsp/imx8/imx8mp/mainline-head.rst [hunk 2]
```diff
@@ -208,7 +208,385 @@ select the phyCORE-|soc| default bootsource.
...
+.. tip::
+
+   A working network is necessary! |ref-setup-network-host|
+
+*  Load your image via network to RAM:
+
+   .. code-block::
+      :substitutions:
+
+      u-boot=> tftp ${loadaddr} |yocto-imagename|-|yocto-machinename|.wic
```
Comment (tremmet) [inline source/bsp/imx8/imx8mp/mainline-head.rst:260]: when using dynamic ips. Do we need to use the dhcp instead of tftp?

Comment (BHahn42618) [reply source/bsp/imx8/imx8mp/mainline-head.rst]: Yes. Done.

File: source/bsp/imx8/imx8mp/mainline-head.rst [hunk 3]
```diff
@@ -208,7 +208,382 @@ select the phyCORE-|soc| default bootsource.
...
+.. tip::
+
+   A working network is necessary! |ref-setup-network-host|
+
+*  Load your image via network to RAM:
+
+   .. code-block::
+      :substitutions:
+
+      u-boot=> dhcp ${loadaddr} |yocto-imagename|-|yocto-machinename|.wic
```
Comment (ymoog) [inline source/bsp/imx8/imx8mp/mainline-head.rst:255]: ${loadaddr} is not necessary.

**Review by ymoog — APPROVED** [review]

**Review by tremmet — APPROVED** [review]

**Review by tremmet — APPROVED** [review]

**Pull request merged**